layers/+vim/evil-better-jumper/README.org

```diff
@@ -19,7 +19,17 @@ locations.
 * Install
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =evil-better-jumper= to the existing =dotspacemacs-configuration-layers=
-list in this file.
+list in this file. If =C-i= inserts an indent instead of
+=better-jumper-jump-forward=, please use =M-x describe-variable= to check the value
+of the symbol =dotspacemacs-distinguish-gui-tab= and ensure it is set to true in
+the dotspacemacs file.
+
+#+begin_src emacs-lisp
+  (defun dotspacemacs/user-config ()
+    ;; ...
+    (setq dotspacemacs-distinguish-gui-tab t)
```

Comment:
Doesn't look quite right.

https://github.com/syl20bnr/spacemacs/blob/a26121db4f60d49dd0ccb0bcf4a4fc538a11bc67/core/templates/.spacemacs.template#L275

Reply:
Yeah I do my best to keep an eye on .spacemacs.template to keep it in sync with my own dotfile, but I have to manually start a diff.

Corrected snippet to its proper location, user-init